torch/_prims/context.py

```diff
@@ -243,5 +249,7 @@ def __torch_function__(
         # First we intercept calls for nvfuser-specific prims bypassing generic torch._refs
         if self._is_var_mean(orig_func):
             return torch.ops.nvprims.var_mean(*args, **kwargs)
+        if self._is_rand_like(orig_func):
+            return torch.ops.nvprims.rand_like(*args)
```

Comment:
So all keyword arguments (dtype, layout, device, pin_memory, memory_format) are ignored. Should we give at least a warning in this case (`len(kwargs) > 0`)?

Reply:
Added a warning.